### ISSUE TITLE
Sub-Action menus

### ISSUE THREAD

**Issue by CoderBrandon** [kivutar/tethical#16]
Right now when choosing action it defaults to a standard attack. We need to create the Sub-Action menu to allow multiple levels of action categories and actions. The data of available actions would need to be pulled from the control panel output.

So the drilldown would be: Action > (Attack, SubAction1, SubAction2) > (SubAction Available Actions)

We could default this to the FFT method for now but would eventually be neat to allow multilevel SubAction groups, ie:

Action > (Attack, SubAction1, SubAction2) > (SubSubAction1, SubSubAction2) > (SubActionN Available Actions)

**Comment by kivutar** [comment]
Yes, the client should receive a data structure that describe the full menu tree from the server. And as you said, the server should get these data from the database.
